src/odemis/gui/comp/miccanvas.py

```diff
@@ -650,6 +650,36 @@ def fit_view_to_content(self, recenter=None):
             if recenter:
                 self.view.view_pos.value = self.requested_phys_pos
 
+    def fit_to_bbox(self, bbox):
+        """
+        Zoom in to the bounding box and recenter. Same as fit_view_to_content, but with explicit bounding box,
+        so we can zoom to a specific position.
+        bbox (None or 4 floats): bounding box to be shown, defined as minx, miny,
+           maxx, maxy positions in m.
+        """
+        if bbox[0] is None:
```

Comment:
bbox seems to be defined as a tuple as 4 None's at first.
Shouldn't it be checked if all the values are not None?

Reply:
Good catch. Actually for this function, I think it makes sense to just assume we pass a correct bbox. Just I'll simplify the code.